src/types.ts

```diff
@@ -0,0 +1 @@
+export * from "./types.deno.ts";
```

Comment:
This file is unnecessary.

Reply:
From what point of view?

Reply:
Deno users can import directly from `src/types.deno.ts`.

Reply:
Wouldn't you consider that to be a leaky abstraction over the platform independency?

Reply:
See https://github.com/grammyjs/grammY/pull/166#issuecomment-1126811724 below